app/src/main/AndroidManifest.xml

```diff
@@ -40,7 +40,7 @@
             android:name=".login.LoginActivity"
```

Comment:
> One thing that still seems a bit off are the different colors of the action bar and the navigation drawer because of the changed colorPrimary in https://github.com/gotify/android/pull/292.

I'd be okay with reverting this back to the old one. Tho having a different color for the top bar and the navigation drawer isn't that bad, because it adds a little contrast, what's your opinion on this?

Reply:
I'll leave that up to you.  
We can leave it as is.

Reply:
oke, then let's leave it as it. Less effort (:.